examples/cua/legacy_report_fetcher/portal/server.py

```
@@ -0,0 +1,296 @@
+
+import argparse
+import csv
+import html
+import io
+from datetime import date, timedelta
+from http import HTTPStatus
+from http.server import BaseHTTPRequestHandler, ThreadingHTTPServer
+from urllib.parse import parse_qs, urlparse
+
+USERNAME = "ops@acme.test"
+PASSWORD = "demo-password"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "PASSWORD"

(S105)